### PR TITLE
IGVF-724-create-Prediction-File-Set

### PR DESCRIPTION
**Pull request by iyoungworth**

### CODE REVIEW

File: src/igvfd/schemas/prediction.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
+    "title": "Prediction",
+    "description": "Schema for submitting a file set of prediction files generated by CAMP.",
```
Comment (gabdank) [inline src/igvfd/schemas/prediction.json:3]: Not sure we want to use abbreviations like CAMP. 

Maybe just "set of computational predictions"?

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/igvfd/searches/configs/prediction.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+from snovault.elasticsearch.searches.configs import search_config
+
+
+@search_config(
+    name='Prediction'
+)
+def cell_line():
```
Comment (gabdank) [inline src/igvfd/searches/configs/prediction.py:7]: not a cell line

**Review by gabdank — CHANGES_REQUESTED** [review]
not a cell line in search config

File: src/igvfd/searches/configs/prediction.py (hunk 2 above)
Comment (iyoungworth) [reply src/igvfd/searches/configs/prediction.py:7]: Same copy paste error as in auxiliary set, sorry about this. Fixed.

**Review by gabdank — DISMISSED** [review]

**Review by gabdank — APPROVED** [review]

**Pull request merged**